enterprise/backend/src/metabase_enterprise/serialization/v2/extract.clj

```diff
@@ -62,17 +62,22 @@
     (eduction (map #(serdes/extract-all % extract-opts)) cat (model-set opts))))
 
 ;; TODO Properly support "continue" - it should be contagious. Eg. a Dashboard with an illegal Card gets excluded too.
-(defn- descendants-closure [targets]
+(defn- chase-for [targets func]
```

Comment:
awesome, this feels like the right abstraction. can you come up with a better name than `func`? This is graph traversal, so we should keep to standard names like `get-adjacent-items`